### ISSUE TITLE
Decide which Unconf branded swag we provide next year

### ISSUE THREAD

**Issue by alster-wasser** [rubyunconfeu/unconf2020#10]
+ T-Shirt
+ bag (gym bag again?)
+ speaker gifts? will be hard to top last year's

**Comment by emrox** [comment]
Ideas:
- Jacket instead of Shirt? or maybe a Hoodie?
- normal bag instead of a gym bag?

**Comment by alster-wasser** [comment]
Let's run some polls on twitter about this next week, see if there is any unexpected consensus

**Comment by ferdinandtrenkel** [comment]
Tote/Shopping bag?
Waistbag?
Pencilcase?
Hodded zipper?
Rain poncho?
Credit card holder?
Jogger pant?

**Comment by alster-wasser** [comment]
As a pen hoarder, I love the pencilcase suggestion! 

Like the waistbag suggestion too, it might be a bit small to fit sponsor swag BUT it could also be an incentive for us to keep plastic/paper trash to a minimum so everything fits.

**Comment by alster-wasser** [comment]
Rain poncho is a great suggestion in Hamburg!! Or branded umbrellas.

**Comment by otaznik-net** [comment]
unconf themed "football scarf"? not too much into the football, but the warmth from towel around neck was cool, and also the ease to identify other attendees was nice.

**Comment by ferdinandtrenkel** [comment]
> unconf themed "football scarf"? not too much into the football, but the warmth from towel around neck was cool, and also the ease to identify other attendees was nice.

Superb idea!

**Comment by alster-wasser** [comment]
Agreed, love that idea! June might be a bit warm for a scarf, but we'll see.
